libs/community/langchain_community/document_loaders/athena.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+import io
+import json
+import time
+from typing import Any, Dict, Iterator, List, Optional, Tuple
+
+import pandas as pd
```

Comment:
pandas isn't a required dependency so it shouldn't be imported at the top level. should instead import in the functions that use it

Reply:
Thanks for the feedback. 
I Have made this change